docs/design/utilities.rst

```diff
@@ -0,0 +1,212 @@
+Utilities
+===============================
+
+Introduction
+------------
+
+Mautic offers a set of CSS utility classes designed to facilitate efficient implementation of flexible layouts using CSS Flexbox. These utilities provide a systematic approach to constructing responsive and dynamic new areas for its user interface, enabling you to rapidly prototype and build complex layouts while minimizing custom CSS.
+
+The utility classes seek to encompass a great spectrum of Flexbox properties, including:
+
+- Display settings
+- Flex direction
+- Wrapping behavior
+- Justification and alignment
+- Spacing and gap management
+- Item ordering
+
+Key Aspects
+^^^^^^^^^^^^
+
+1. **Abbreviated Nomenclature**: Class names follow a logical, short and easy-to-remember convention.
+2. **Most Common Uses Coverage**: Utilities address the full range of Flexbox properties and behaviors.
+3. **Responsive Design**: Includes variations for different viewport sizes, enabling fine-grained control over layout across devices.
+4. **Composability**: Classes can be combined to create sophisticated layout patterns.
+5. **Consistent Spacing**: Standardized options for managing padding, margin and gap properties.
+
+This documentation is structured to provide a clear understanding of each utility class, its function, and its application. It includes practical examples and best practices for combining utilities to achieve complex layout requirements.
+
+The following sections detail the individual utility classes, their usage, and advanced implementation strategies. This resource is intended for front-end developers seeking to leverage Flexbox efficiently in their projects.
+
+
+Abbreviation Keys
+----------------
+
+- **d**: display
+- **fd**: flex-direction
+- **fw**: flex-wrap
+- **jc**: justify-content
+- **ai**: align-items
+- **ac**: align-content
+- **as**: align-self
+- **fg**: flex-grow
+- **fs**: flex-shrink
+- **fb**: flex-basis
+- **fo**: order
+
+Flex Container Properties
```

Comment:
This list should probably be broken down by theme. Right now it looks like a lot of options but every type of change has several variants.
So basically, all the option classes that have the same prefix, could be under a specific heading, making things more readable and easier to find. Something like this:
```
### direction
.fd-row
…
### wrap
.fw-wrap
…
```